app.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Dobra praksa: Celotno kodo zavij v IIFC (immediately invoked function call). To je na kratko funkcija , ki se takoj sprozi. To je dobra praksa zato, da imas svojo aplikacijo v svojem scope-u (konkretno, v scope-u te funkcije) in vrednosti ne uhajajo v global scope. 

Tore:
`(function(){  //vsa tvoja koda...  })();`